how-to/software/upgrade-your-release.md

```diff
@@ -7,10 +7,24 @@ We recommend running a Long Term Support (LTS) release as it provides 5 years of
 
 After the initial standard support period ends for an LTS release, an extended maintenance period is available via an [Ubuntu Pro subscription](http://ubuntu.com/pro), which provides coverage for an additional five years and is available for free on up to five machines. Find out more about the [release lifecycle and support period](https://ubuntu.com/about/release-cycle) for your release.
 
+## Update vs Upgrade vs Release-Upgrade
+
+```{note}
```

Comment:
I think this can be a normal paragraph, and not a note. It looks better, since the text follows a new title.